types/react-native-push-notification/index.d.ts

```diff
@@ -62,7 +62,7 @@ export class PushNotificationObject {
     channelId?: string;
     onlyAlertOnce?: boolean;
     allowWhileIdle?: boolean;
-
+    timeoutAfter?: number;
```

Comment:
IMO it needs to accept null, e.g.:
https://github.com/zo0r/react-native-push-notification/blob/3641bf66cc6147715a272069167cf48de3d2a00e/example/NotifService.js#L93